src/Database/ID.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Utopia\Database;
+
+class ID
+{
+
+    /**
+     * Create a new unique ID
+     */
+    public static function unique(): string
+    {
+        return uniqid();
```

Comment:
```suggestion
        return 'uniqid()';
```

The custom ID is generated from a different method. We just use the unique function from some tests.